File: Examples/VisualBasic/ApplyLicense/ApplyLicenseByEmbeddedResource.vb
Imports Aspose.Note
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace ApplyLicense
    Public Class ApplyLicenseByEmbeddedResource
        Public Shared Sub Run()
            ' ExStart:ApplyLicenseByEmbeddedResource        
            ' Instantiate the License class
            Dim license As New Aspose.Note.License()

            ' Pass only the name of the license file embedded in the assembly
            license.SetLicense("Aspose.Note.lic")
            ' ExEnd:ApplyLicenseByEmbeddedResource
        End Sub
    End Class
End Namespace

File: Examples/CSharp/ApplyLicense/ApplyLicenseByEmbeddedResource.cs
using Aspose.Note;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Aspose.Note.Examples.CSharp.ApplyLicense
{
    public class ApplyLicenseByEmbeddedResource
    {
        public static void Run() 
        {
            // ExStart:ApplyLicenseByEmbeddedResource
            // ExFor:License
            // ExFor:License.SetLicense(System.String)
            // ExSummary:Shows how to load a license for Aspose.Note from embedded file resource.

            // Instantiate the License class
            Aspose.Note.License license = new Aspose.Note.License();

            // Pass only the name of the license file embedded in the assembly
            license.SetLicense("Aspose.Note.lic");
            
            // ExEnd:ApplyLicenseByEmbeddedResource
        }
    }
}
